src/eventshub/src/App.tsx

```diff
@@ -1,18 +1,13 @@
-import { useState } from 'react'
+import { SearchTab } from './component/SearchTab';
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <>
-      <h2 className="text-digital-red">Vite + React</h2>
-      <div>
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-      </div>
-    </>
-  )
+    <div>
+      <h2>Vite + React</h2>
+      <SearchTab searchIndex='SERENE ALL - appEb3LGlZS9OfNrK - Web Only' />
```

Comment:
I'll focus on building out the multi-searchIntext in https://stanfordits.atlassian.net/browse/EIRH-50 🤔 Otherwise I've put the `Web Only` index here for our testing purposes.